sashimi/state.py

```diff
@@ -224,16 +225,20 @@ def calculate_calibration(self):
 
 
 def get_voxel_size(
-    scanning_settings: ZRecordingSettings,
+    scanning_settings: Union[ZRecordingSettings, SinglePlaneSettings],
     camera_settings: CameraSettings,
 ):
-    scan_length = (
-        scanning_settings.piezo_scan_range[1] - scanning_settings.piezo_scan_range[0]
-    )
 
     binning = int(camera_settings.binning)
 
-    inter_plane = scan_length / scanning_settings.n_planes
+    if isinstance(scanning_settings, SinglePlaneSettings):
+        inter_plane = 1
+    else:
+        scan_length = (
+            scanning_settings.piezo_scan_range[1]
+            - scanning_settings.piezo_scan_range[0]
+        )
+        inter_plane = scan_length / scanning_settings.n_planes
```

Comment:
Inside `get_voxel_size`:
checks if the scanning_settings are single_plane_settings and if so sets the z_voxel size to 1